### ISSUE TITLE
ACCESSIBILITY- lighthouse and wave

### ISSUE THREAD

**Issue by laurelbonal** [laurelbonal/overlook-hotel#9]
You must score as close to 100% as possible with the Lighthouse and WAVE accessibility audits. Be prepared to explain any accessibility audits your application is failing.
